src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/MemberNameCheck.java

```diff
@@ -63,6 +63,7 @@
  *   int num3; // OK
  *   private int num4; // OK
  *
+ *   public static final int CONSTANT = 1; // ignored: not an instance variable
```

Comment:
I am not seeing static non-final, and non-static final examples. Lets add them just so we can be extra clear.

Reply:
done